package.json

```diff
@@ -13,7 +13,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "license": "MIT",
-  "main": "dist/cjs/index.js",
+  "main": "dist/esm/index.js",
```

Comment:
@kikuomax are the main and module swapped in our original package.json file?

Reply:
@wesdevpro No, `main` should point at the CJS entry for compatibility with legacy Node.js and bundlers.